src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java

```diff
@@ -48,7 +49,7 @@ private static boolean isMatchedAbbreviated(String name, Abbreviation abbreviati
      * Letters) or its abbreviated form (e.g. Phys. Rev. Lett.).
      */
     public boolean isKnownName(String journalName) {
-        String journal = journalName.trim();
+        String journal = journalName.trim().replaceAll(Matcher.quoteReplacement("\\&"), "&");
```

Comment:
I think, there is the possibility to pre-compile the Regex. Was it with the `Pattern` class?